Instagram/View Controllers/HomeViewController.m

```diff
@@ -12,6 +19,39 @@ @implementation HomeViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    
+    [self fetchPosts];
+    
+    self.refreshControl = [[UIRefreshControl alloc] init];
+    [self.refreshControl addTarget:self action:@selector(fetchPosts) forControlEvents:UIControlEventValueChanged];
+    [self.tableView insertSubview:self.refreshControl atIndex:0];
+}
+
+- (void)viewWillAppear:(BOOL)animated {
+    NSIndexPath *selectedIndPath = self.tableView.indexPathForSelectedRow;
```

Comment:
plz avd abbrvs like ths. They don't make anything easier to read and actually make it much harder to read. Especially something like this where it's not a common term to abbreviate.